app/broadcast_message/utils.py

```diff
@@ -133,9 +134,7 @@ def _create_broadcast_event(broadcast_message):
             message_type=msg_types[broadcast_message.status],
             transmitted_content={"body": broadcast_message.content},
             transmitted_areas=broadcast_message.areas,
-            # TODO: Probably move this somewhere more standalone too and imply that it shouldn't change. Should it
-            # include a service based identifier too? eg "flood-warnings@notifications.service.gov.uk" or similar
-            transmitted_sender="notifications.service.gov.uk",
+            transmitted_sender=SENDER,
```

Comment:
This doesn't appear to be used at all, so subtly changing it to the constant _shouldn't_ matter. The CBC proxy/utils XML methods use `SENDER` during the XML generation anyway.